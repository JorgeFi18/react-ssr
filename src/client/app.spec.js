import React from 'react';
import renderer from 'react-test-renderer';
import App from './app';

describe('App component', ()=>{
    it('Should match with snapshot', () => {
        const component = renderer.create(
            <App />
        )

        const tree = component.toJSON();

        expect(tree).toMatchSnapshot();
    });
});