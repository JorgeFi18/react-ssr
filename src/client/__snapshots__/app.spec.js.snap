// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Should match with snapshot 1`] = `
<div>
  <header
    className="header"
  >
    <figure
      className="header__logo"
    >
      Marvel Api
    </figure>
  </header>
  <div
    className="searcher"
  >
    <input
      className="searcher__input"
      name="search"
      onChange={[Function]}
      placeholder="Search for your character"
      type="search"
    />
  </div>
  <div
    className="filter"
  >
    <p
      className="filter__label"
    >
      Filter by:
    </p>
    <select
      className="filter__input"
      onChange={[Function]}
      value="name"
    >
      <option
        value="name"
      >
        Name (ASC)
      </option>
      <option
        value="modified"
      >
        Date (ASC)
      </option>
      <option
        value="-name"
      >
        Name (DESC)
      </option>
      <option
        value="-modified"
      >
        Date (DESC)
      </option>
    </select>
  </div>
  <div
    className="heros-list"
  />
</div>
`;
